data/src/test/java/com/linkedin/data/schema/TestPdlBuilder.java

```diff
@@ -37,31 +39,39 @@
   private static Object[][] propertiesMapProvider()
   {
     DataMap properties1 = new DataMap();
-    properties1.put("empty", new DataMap());
+    properties1.put("empty", new DataList(Arrays.asList(1, 2, 3)));
     DataMap properties2 = new DataMap();
     properties2.put("validate", properties1);
     return new Object[][]
         {
             {
               properties1,
-              "@empty = {}\n"
+              "@empty = [ 1, 2, 3 ]\n",
+              "@empty=[1,2,3]"
             },
             {
               properties2,
-              "@validate.empty = {}\n"
+              "@validate.empty = [ 1, 2, 3 ]\n",
+              "@validate.empty=[1,2,3]"
             }
         //TODO Add test case for multiple properties in a map level once iteration logic is fixed to be deterministic
```

Comment:
Can you add the testcases you used for testing, but comment them out for now. It would be easier to just enable those tests later.

Reply:
done